rows.go

```diff
@@ -2,74 +2,77 @@ package clickhouse
 
 import (
 	"database/sql/driver"
+	"encoding/csv"
 	"io"
 	"reflect"
 	"time"
 )
 
-type textRows struct {
-	columns []string
-	types   []string
-	data    []byte
-	decode  decoder
-}
+func nextTextRows(c *conn, body io.ReadCloser, location *time.Location, useDBLocation bool) (*streamingRows, error) {
```

Comment:
It does need all this changes? 
I prefer to save original names of functions and structures to minimize number of changes.
`func newTextRows`, `type textRows`

Reply:
agreed! Fixed!